AzureSpatialAnchorsSample/ProjectSettings/ProjectSettings.asset

```diff
@@ -770,7 +770,7 @@ PlayerSettings:
       SpatialPerception: True
       InternetClient: True
       WebCam: True
-      Microphone: False
+      Microphone: True
```

Comment:
Does the ASA sample use this microphone capability?

Reply:
there was a warning that microphone capability was required. It may be for something build into MRTK? I'd also be happy to keep this off and ignore the warning if it's not really needed

Reply:
ahh, good to know! if it's on for a reason, all good; I just wasn't expecting it at first